rpcs3/Emu/RSX/VK/VKHelpers.cpp

```diff
@@ -71,6 +71,11 @@ namespace vk
 
 		vk::get_upload_heap()->destroy();
 
+		for (const auto& p : g_compute_tasks)
+		{
+			p.second->destroy();
```

Comment:
Automatically handled in compute_task destructor. If it didn't you would get a driver crash every time the emulator was closed.